<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bomberman_dialog</class>
 <widget class="QDialog" name="Bomberman_dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Title</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resurse.qrc">
    <normaloff>:/bomb/img/Bomb.jpg</normaloff>:/bomb/img/Bomb.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>370</width>
     <height>118</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/plain/img/plain color.jpg)</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_username">
          <property name="styleSheet">
           <string notr="true">border-image:transparent;</string>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:transparent;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_password">
          <property name="styleSheet">
           <string notr="true">border-image:transparent;</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="styleSheet">
           <string notr="true">border-image:transparent;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QCommandLinkButton" name="commandLinkButton_login">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
      <property name="icon">
       <iconset resource="Resurse.qrc">
        <normaloff>:/login/img/login.png</normaloff>:/login/img/login.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_signup">
        <property name="text">
         <string>Register</string>
        </property>
        <property name="icon">
         <iconset resource="Resurse.qrc">
          <normaloff>:/signup/img/signup.png</normaloff>:/signup/img/signup.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>-11</y>
     <width>711</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/title/img/Bomberman Title.jpg);</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton_exit</zorder>
 </widget>
 <resources>
  <include location="Resurse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>Bomberman_dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
